Cloudinary/Operation/02_CloudinaryMapping.md

```diff
@@ -0,0 +1,135 @@
+
+  > [Info] Depending on the selected row, the fields displayed in the *Settings* section differ:
+  - If the selected row contains only a destination attribute, the destination attribute as well as the *Extension* and *Override change tracking mode* drop-down lists are displayed.
+  - If the selected row contains an attribute mapping with source, destination attribute and the extension, the destination attribute as well as the *Source attribute*, *Extension* and *Override change tracking mode* drop-down lists are displayed. The *Source attribute*, *Extension* and *Override change tracking mode* drop-down lists are preset. If the selected extension requires further configuration settings, the *Configuration* section is displayed below the *Settings* section.  
+
+
+3. Select the appropriate option in the *Extension* drop-down list in the *Settings* section. The following extensions are available to map a Cloudinary source attribute with:
+  - **Copy cloudinary image value**: Copy the value from the Cloudinary image attribute and map it to another Cloudinary image attribute. This extension is only available for a destination attribute with the *Cloudinary image* data type.
+  - **Cloudinary to string**: Map a Cloudinary image to a string attribute. If desired, the image transformation can also be mapped to the destination attribute. This extension is only available for a destination attribute with the *String* data type.
+
+  The *Configuration* section is displayed below the *Settings* section.
+
+  > [Info] For detailed information about the ETL extensions, see [ETL extensions list](../../DataHub/UserInterface/03_ETLExtensions.md).   
+  For detailed information about the data types, see [Data type list](../../PIM/UserInterface/04_DataTypeList.md).
+
+
+4. Select the appropriate tracking mode in the *Override change tracking mode* drop-down list. By default, the **Use default** option is preselected. The following modes are available:
+  - **Use default**: ---
+  - **Automatic**: The initial mapping is automatically applied. All value changes or reruns are applied automatically in the mapping.  
+  - **Semi-automatic**: The initial mapping is automatically applied. Values changes or reruns are applied after confirmation in the *Omni-Channel* module.
+  - **Semi-automatic, changes must be confirmed by another user**: The initial mapping as well as values changes or reruns are applied after confirmation by another user in the *Omni-Channel* module.
+  - **Manual**: The initial mapping is automatically applied. Afterwards, the changes in the mapping must be triggered manually to be applied.
+
+
+5. Select the Cloudinary attribute(s) that you want to assign to the destination attribute. For the *Copy cloudinary image value* ETL extension, you have to select a single Cloudinary source attribute. For the *Cloudinary to string* ETL extension, you can select one to four Cloudinary source attributes and, if desired, add a transformation.
```

Comment:
```suggestion
5. Select the Cloudinary attribute(s) that you want to assign to the destination attribute. For the *Copy Cloudinary image value* ETL extension, you have to select a single Cloudinary source attribute. For the *Cloudinary to string* ETL extension, you can select one to four Cloudinary source attributes and, if desired, add a transformation.
```

Reply:
Vorschlag (finde besser lesbar):

Select the Cloudinary attribute(s) that you want to assign to the destination attribute: 

- For the *Copy Cloudinary image value* ETL extension, you have to select a single Cloudinary source attribute. 

- For the *Cloudinary to string* ETL extension, you can select one to four Cloudinary source attributes and, if desired, add a transformation.